yaksh/templates/yaksh/monitor.html

```diff
@@ -68,24 +68,39 @@
         {% if quiz %}
         {% if papers %}
         <div class="row">
-        <div class="card col-md-4">
-            <div class="row">
-                <div class="col"><b>Course Name:<br>Quiz Name:<br>Number of papers:<br>Papers completed:<br>Papers in progress:</b>
-                </div>
-                <div class="col">                
-                     {{ course.name }}<br>
-                     {{ quiz.description }}<br>
-                    {{ papers|length }}<br>
-                    {% completed papers as completed_papers %}
-                     {# template tag used to get the count of completed papers #}
-                      <b class="yakshred"> {{ completed_papers }} </b><br>
-                    {% inprogress papers as inprogress_papers %}
-                     {# template tag used to get the count of inprogress papers #}
-                     <b class="yakshred">  {{ inprogress_papers }} </b>
-                </div>
+        <div class="card col-md-10">
+            <div class = "table-responsive"">
+                <table class = "table">
+                    <tr>
+                        <td><b>Course Name:&nbsp</b></td>
+                        <td>{{course.name}}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Quiz Name:&nbsp</b></td>
+                        <td>{{quiz.description}}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Number of papers: &nbsp</b></td>
+                        <td>{{papers|length}}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Papers Completed: &nbsp</b></td>
+                        <td>
+                            {% completed papers as completed_papers %}
+                            <b class = "yakshred">{{completed_papers}}</b>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><b>Papers in progress: &nbsp</b></td>
+                        <td>
+                            {% inprogress papers as inprogress_papers %}
+                            <b class="yakshred">  {{ inprogress_papers }} </b>
```

Comment:
since you are modifying this can we have different color and not same as above.
My suggestion completed can be green and in-progress can be red.